src/codemodder/codemods/base_codemod.py

```diff
@@ -31,6 +33,15 @@ class Metadata:
     review_guidance: ReviewGuidance
     references: list[Reference] = field(default_factory=list)
     description: str | None = None
+    tool: ToolMetadata | None = None
+
+
+@dataclass
+class ToolMetadata:
```

Comment:
since this is codetf related should it not be a pydantic class?

Reply:
Good question: this is sort of intermediate metadata that isn't directly used for CodeTF generation so I think `dataclass` is okay.